daemon/container_operations_unix.go

```diff
@@ -353,6 +355,20 @@ func killProcessDirectly(cntr *container.Container) error {
 				logrus.Debug(e)
 				return e
 			}
+
+			// In case there were some exceptions(e.g., state of zombie and D)
+			if system.IsProcessAlive(pid) {
+
+				// Since we can not kill a zombie pid, add zombie check here
+				isZombie, err := system.IsProcessZombie(pid)
+				if err != nil {
+					logrus.Warningf("Container %s state is invalid", stringid.TruncateID(cntr.ID))
```

Comment:
Oh; didn't even know there was a `Warningf` 🤔 - can you use `Warnf` instead? It looks to be exactly the same, but it's what we use everywhere else

```suggestion
					logrus.Warnf("Container %s state is invalid", stringid.TruncateID(cntr.ID))
```

Reply:
done